### PR TITLE
List numbers

### LINKED ISSUE #95: Implement `listNumbers`

### PR DESCRIPTION
**Pull request by judy2k**
Resolves #95, implementing `NumbersClient.listNumbers()`

## Contribution Checklist
* [x] Unit tests!
* [x] Updated [CHANGELOG.md](CHANGELOG.md)
* [x] My name is in [CONTRIBUTORS.md](CONTRIBUTORS.md)

### CODE REVIEW

File: src/main/java/com/nexmo/client/numbers/ListNumbersResponse.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nexmo.client.NexmoUnexpectedException;
+import com.nexmo.client.voice.Call;
+import com.nexmo.client.voice.VoiceClient;
+
+import java.io.IOException;
+
+/**
+ * Response if DTMF tones were successfully sent to an active {@link Call}.
```
Comment (ChrisGuzman) [inline src/main/java/com/nexmo/client/numbers/ListNumbersResponse.java]: Is this copy and pasted from another class?

Comment (judy2k) [reply src/main/java/com/nexmo/client/numbers/ListNumbersResponse.java]: I'm so stupid! I found this in one of my other copy-paste efforts, and fixed it, but didn't go back to find the others I'd missed.

**Comment by judy2k** [comment]
@ChrisGuzman Another one that should be more-or-less ready to go now.

**Comment by ChrisGuzman** [comment]
🚢 🇮🇹

**Comment by judy2k** [comment]
These changes have been merged into #114

**Pull request closed without merging**